src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -1251,7 +1276,7 @@ private void BuildChunksCache()
                 ref var compounds = ref chunk.Get<CompoundStorage>();
 
                 if (!compounds.Compounds.HasAnyCompounds())
-                    continue;
+                    return;
```

Comment:
I think this return is a mistake here as it stops processing entities on the first chunk that has no compounds. There'd be pretty wild AI bug reports if randomly the AI was unable to see most chunks.

Reply:
Note that for some reason microbes don't work with chunks and thus siderophore doesn't too. I added it "blindly" (without testing) so it's done but chunks for microbe must be fixed first to see if it works.

Reply:
I don't get what you are referring to here... Could you explain?

Reply:
Microbes don't see chunks, in short. Either something with cache or further in code.

Reply:
This being in the cache code definitely breaks the cache as the first chunk without compounds causes all further chunks to never be put in the cache (due to the return).

Reply:
Microbes don't care for regular microbial chunks either on master

Reply:
What do you mean by don't care? Did you check with a debugger to see where the microbes reject trying to pursue a chunk? I'd assume at least that the cache should now work without the `return`.

Reply:
I don't have debugger set up, at least yet (if you mean the Thrive debugger, I don't even know how to use my IDE debugger). I'll maybe try to find the cause but no promises.

Reply:
The other option than a debugger is putting a ton of print statements all over the place to try to figure out where the AI chunk checks fail. As it is a pretty complex situation it's going to be much harder to try to debug without a proper debugger, I recommend taking a bit of time to set a C# debugger up for use with Thrive.